erts/emulator/sys/unix/sys_signal_stack.c

```diff
@@ -273,10 +273,9 @@ static void sys_sigaltstack(void *ss_sp) {
 /*
  * Set up alternate signal stack for an Erlang process scheduler thread.
  */
+static char sys_signalstack[SIGSTKSZ];
 void sys_thread_init_signal_stack(void) {
-    /* This will never be freed. */
-    char *stack = malloc(SIGSTKSZ);
-    sys_sigaltstack(stack);
+    sys_sigaltstack(sys_signalstack);
 }
```

Comment:
This will lead to multiple threads (e.g. all schedulers) sharing the same signal stack which may cause a crash